src/lib/openapi/spec/profile-schema.ts

```diff
@@ -6,22 +6,29 @@ export const profileSchema = {
     $id: '#/components/schemas/profileSchema',
     type: 'object',
     additionalProperties: false,
+    description: 'User profile overview',
     required: ['rootRole', 'projects', 'features'],
     properties: {
         rootRole: {
+            description:
```

Comment:
the same note about $refs goes here. I might be wrong though, but I think we checked this?